modules/manage/pages/manage-security/configure-server-certificates.adoc

```diff
@@ -640,18 +672,25 @@ Consequently:
 This is because a node can be joined to a cluster only prior to the node's provisioning; while certificate-deployment can only occur _subsequent to_ the node's provisioning.
```

Comment:
>  No node can be _joined_ to a cluster.
> This is because a node can be joined to a cluster only prior to the node's provisioning; while certificate-deployment can only occur _subsequent to_ the node's provisioning.

I can't understand this sentence. "No node can be joined to a cluster" sounds like it's impossible to add a node to a cluster. I suggest we modify it.